sqlstore/src/main/resources/db/migration/V5201__PacBio_library_options.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO LibraryType (description, platformType, archived) VALUES ('Whole Genome', 'PACBIO', 0);
+
+INSERT INTO LibrarySelectionType (name, description) VALUES ('SageHLS', 'Sage High Molecular Weight Library System'), ('BluePippin', 'Sage BluePippin size selection'), ('None', 'None');
```

Comment:
Is 'Sage BluePippin size selection' distinct from the existing 'size fractionation' option?

'None' should not be an option since the field is nullable